Dockerfile

```diff
@@ -27,3 +27,6 @@ COPY src/ src/
 
 EXPOSE 3000
 CMD [ "yarn", "start" ]
+
```

Comment:
Should we alter the lines at https://github.com/elifesciences/enhanced-preprints-server/blob/article-version-not-found/docker-compose.yaml#L63-L64 to match?